.gitignore

```diff
@@ -25,4 +25,5 @@ rls*.log
 *.bin
 *.iml
 scripts/ci/node-template-release/Cargo.lock
+bin/node-template/Cargo.lock
```

Comment:
As far as I know, we deliberately commit `Cargo.lock` to git to make sure we use specific library versions on all builds. May be not that important for `node-template` though. CC @bkchr

Reply:
Yes, but here in our workspace we only have `Cargo.lock` at the top level. So, this is fine.